README.md

```diff
@@ -54,3 +54,20 @@ Generates html and json reports.
 ### tap reporter
 
 Example script: `automated:watchMe:tap`
+
+### Slack reporter (mocha-ci-slack-reporter)
+
+* Add Slack reporter to the project:
+```
+yarn add mocha-ci-slack-reporter --dev
```

Comment:
Above in ## Example tests we provide commands for npm, probably should do it here as well

Reply:
We should just add this to our package.json, same as for `tap`, `junit` etc.